src/View/mainpage/DeleteROIWindow.py

```diff
@@ -63,12 +62,14 @@ def setup_ui(self, delete_roi_window_instance, regions_of_interest, dataset_rtss
         # Create a label for the tree view with the list of ROIs to keep
         self.delete_roi_window_keep_tree_view_label = QLabel()
         self.delete_roi_window_keep_tree_view_label.setObjectName("DeleteRoiWindowKeepTreeViewLabel")
-        self.delete_roi_window_keep_tree_view_label.setProperty("QLabelClass", ["tree-view-label", "tree-view-label-keep-delete"])
+        self.delete_roi_window_keep_tree_view_label.setProperty("QLabelClass",
```

Comment:
Too long for PEP 8, as is line 226 and 328.

Reply:
fixed